.gitignore

```diff
@@ -1,4 +1,4 @@
-**/build*/
+/build*/
```

Comment:
That double asterisk catched subdirectories, too. This is especially useful for programs inside the `example/` path (and would have avoided things like roboticslab-uc3m/speech@d6ba47b). I wonder what was the reason behind 812f8f2, though.